install_config/persistent_storage/topics/glusterfs_uninstall.adoc

```diff
@@ -0,0 +1,22 @@
+For {gluster-native}, the {product-title} Advanced Installer comes with a
+playbook to uninstall all resources and artifacts from the cluster. To use it,
+provide the original inventory file that was used to install the target instance
+of {gluster-native} and run the following playbook:
+
+----
+ansible-playbook -i <path_to_inventory_file> /usr/share/ansible/openshift-ansible/playbooks/openshift-glusterfs/uninstall.yml
+----
+
+In addition, the playbook supports the use of a variable called
+`openshift_storage_glusterfs_wipe` which, when enabled, will destroy any data on
+the block devices that were used for {gluster} backend storage. It is recommend
+to use this variable like so:
```

Comment:
"It is recommended" is expletive construction. Maybe just "To use the `openshift_storage_glusterfs_wipe` variable:"